src/Files.App/ViewModels/Settings/AdvancedViewModel.cs

```diff
@@ -243,43 +261,29 @@ private async Task ExportSettings()
 			}
 		}
 
-		private bool DetectIsSetAsDefaultFileManager()
+		private static bool DetectIsSetAsDefaultFileManager()
 		{
 			using var subkey = Registry.ClassesRoot.OpenSubKey(@"Folder\shell\open\command");
 			var command = (string?)subkey?.GetValue(string.Empty);
 
 			return !string.IsNullOrEmpty(command) && command.Contains("FilesLauncher.exe");
 		}
 
-		private bool DetectIsSetAsOpenFileDialog()
+		private static bool DetectIsSetAsOpenFileDialog()
 		{
 			using var subkey = Registry.CurrentUser.OpenSubKey(@"SOFTWARE\Classes\CLSID\{DC1C5A9C-E88A-4DDE-A5A1-60F82A20AEF7}");
 
 			return subkey?.GetValue(string.Empty) as string == "FilesOpenDialog class";
 		}
 
-		private bool isSetAsDefaultFileManager;
-		public bool IsSetAsDefaultFileManager
-		{
-			get => isSetAsDefaultFileManager;
-			set => SetProperty(ref isSetAsDefaultFileManager, value);
-		}
-
-		private bool isSetAsOpenFileDialog;
-		public bool IsSetAsOpenFileDialog
-		{
-			get => isSetAsOpenFileDialog;
-			set => SetProperty(ref isSetAsOpenFileDialog, value);
-		}
```

Comment:
Is there a reason you moved these before the constructor?

Reply:
According to the guideline, section 4.

> The following presents how a class structure should be ordered:
> 
> Static fields
> Static properties
> readonly fields
> Fields
> _disposed field (if applicable)
> Service property
> Properties
> UI properties
> ICommand properties
> Constructor { fields injection, initialization of fields, command implementation }
> Command implementation functions (private/protected only, can be virtual but not abstract)
> public (virtual/override) methods (with parameters amount descending)
> 
> protected (virtual/override) methods (with parameters amount descending)
> 
> private methods
> 
> static methods
> abstract methods
> Operators
> Assertions e.g. AssertNotDisposed(), AssertInitialized(), AssertStreamOpened()
> IDisposable.Dispose() function